src/Entities/Pagination.php

```diff
@@ -0,0 +1,118 @@
+<?php
+
+namespace ArrowSphere\PublicApiClient\Entities;
+
+class Pagination
+{
+    protected const PAGINATION_PER_PAGE = 'per_page';
+    protected const PAGINATION_CURRENT_PAGE = 'current_page';
+    protected const PAGINATION_TOTAL_PAGE = 'total_page';
+    protected const PAGINATION_TOTAL = 'total';
+    protected const PAGINATION_NEXT_PAGE = 'next';
+    protected const PAGINATION_PREVIOUS_PAGE = 'previous';
+
+    const DEFAULT_PER_PAGE = 15;
+
+    /**
+     * @var int
+     */
+    private $perPage;
+
+    /**
+     * @var int
+     */
+    private $currentPage;
+
+    /**
+     * @var int|null
+     */
+    private $totalPage;
+
+    /**
+     * @var int|null
+     */
+    private $total;
+
+    /**
+     * @var string
+     */
+    private $nextPage;
+
+    /**
+     * @var string
+     */
+    private $previousPage;
+
+    public function __construct(array $data)
+    {
+        $this->perPage = $data[self::PAGINATION_PER_PAGE] ?? self::DEFAULT_PER_PAGE;
```

Comment:
I don't think you should handle default values
if you don't have "per page" information, what says it's 15?